static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -121,6 +126,7 @@ function Onboarding(props: Props) {
       <SentryDocumentTitle title={stepObj.title} />
       <Header>
         <LogoSvg />
+        {stepId !== 'welcome' && <StyledStepper steps={ONBOARDING_STEPS.slice(1)} currentStepId={stepId} onClick={goToStep} />}
```

Comment:
@Neo-Zhixing could we go off the step index instead of the name? So show it we're passed the 0th step.

Reply:
they're pretty much equivalent and one could also argue that stepId == 'welcome' because it shows we're passing the welcome step.